catalogue/aarch64-MTE/shelf.py

```diff
@@ -58,4 +58,1097 @@
+    "tests/LB+dsb.sypt+datatp.litmus",
+    "tests/LB+dsb.sypt+datatt.litmus",
+    "tests/LB+dsb.sypt+dsb.sytp.litmus",
+    "tests/LB+dsb.sypt+dsb.sytt.litmus",
+    "tests/LB+dsb.sypts.litmus",
+    "tests/LB+dsb.sys.litmus",
+    "tests/LB+dsb.sytp+addr.litmus",
+    "tests/LB+dsb.sytp+addrpt.litmus",
+    "tests/LB+dsb.sytp+addrtp.litmus",
+    "tests/LB+dsb.sytp+addrtt.litmus",
+    "tests/LB+dsb.sytp+data.litmus",
+    "tests/LB+dsb.sytp+datapt.litmus",
+    "tests/LB+dsb.sytp+datatp.litmus",
+    "tests/LB+dsb.sytp+datatt.litmus",
+    "tests/LB+dsb.sytp+dsb.sytt.litmus",
+    "tests/LB+dsb.sytps.litmus",
+    "tests/LB+dsb.sytt+addr.litmus",
+    "tests/LB+dsb.sytt+addrpt.litmus",
+    "tests/LB+dsb.sytt+addrtp.litmus",
+    "tests/LB+dsb.sytt+addrtt.litmus",
+    "tests/LB+dsb.sytt+data.litmus",
+    "tests/LB+dsb.sytt+datapt.litmus",
+    "tests/LB+dsb.sytt+datatp.litmus",
+    "tests/LB+dsb.sytt+datatt.litmus",
+    "tests/LB+dsb.sytts.litmus",
+    "tests/MP+dmb.st+addr.litmus",
+    "tests/MP+dmb.st+addrpt.litmus",
+    "tests/MP+dmb.st+addrtp.litmus",
+    "tests/MP+dmb.st+addrtt.litmus",
+    "tests/MP+dmb.st+dmb.ld.litmus",
+    "tests/MP+dmb.st+dmb.ldpt.litmus",
+    "tests/MP+dmb.st+dmb.ldtp.litmus",
+    "tests/MP+dmb.st+dmb.ldtt.litmus",
+    "tests/MP+dmb.st+dmb.sy.litmus",
+    "tests/MP+dmb.st+dmb.sypt.litmus",
+    "tests/MP+dmb.st+dmb.sytp.litmus",
+    "tests/MP+dmb.st+dmb.sytt.litmus",
+    "tests/MP+dmb.st+dsb.sy.litmus",
+    "tests/MP+dmb.st+dsb.sypt.litmus",
+    "tests/MP+dmb.st+dsb.sytp.litmus",
+    "tests/MP+dmb.st+dsb.sytt.litmus",
+    "tests/MP+dmb.stpt+addrpt.litmus",
+    "tests/MP+dmb.stpt+addrtp.litmus",
+    "tests/MP+dmb.stpt+addrtt.litmus",
+    "tests/MP+dmb.stpt+dmb.ld.litmus",
+    "tests/MP+dmb.stpt+dmb.ldpt.litmus",
+    "tests/MP+dmb.stpt+dmb.ldtp.litmus",
+    "tests/MP+dmb.stpt+dmb.ldtt.litmus",
+    "tests/MP+dmb.stpt+dmb.sy.litmus",
+    "tests/MP+dmb.stpt+dmb.sypt.litmus",
+    "tests/MP+dmb.stpt+dmb.sytp.litmus",
+    "tests/MP+dmb.stpt+dmb.sytt.litmus",
+    "tests/MP+dmb.stpt+dsb.sy.litmus",
+    "tests/MP+dmb.stpt+dsb.sypt.litmus",
+    "tests/MP+dmb.stpt+dsb.sytp.litmus",
+    "tests/MP+dmb.stpt+dsb.sytt.litmus",
+    "tests/MP+dmb.sttp+addr.litmus",
+    "tests/MP+dmb.sttp+addrpt.litmus",
+    "tests/MP+dmb.sttp+addrtp.litmus",
+    "tests/MP+dmb.sttp+addrtt.litmus",
+    "tests/MP+dmb.sttp+dmb.ld-show.litmus",
+    "tests/MP+dmb.sttp+dmb.ld.litmus",
+    "tests/MP+dmb.sttp+dmb.ldpt.litmus",
+    "tests/MP+dmb.sttp+dmb.ldtp.litmus",
+    "tests/MP+dmb.sttp+dmb.ldtt.litmus",
+    "tests/MP+dmb.sttp+dmb.sy.litmus",
+    "tests/MP+dmb.sttp+dmb.sypt.litmus",
+    "tests/MP+dmb.sttp+dmb.sytp.litmus",
+    "tests/MP+dmb.sttp+dmb.sytt.litmus",
+    "tests/MP+dmb.sttp+dsb.sy.litmus",
+    "tests/MP+dmb.sttp+dsb.sypt.litmus",
+    "tests/MP+dmb.sttp+dsb.sytp.litmus",
+    "tests/MP+dmb.sttp+dsb.sytt.litmus",
+    "tests/MP+dmb.sttp.litmus",
+    "tests/MP+dmb.sttt+addr.litmus",
```

Comment:
This test is not present in sub-directory test, nor in the kind file. We may either delete this line, or keep it. add  the test and a kind for  it. The model gives kind Allow. I attach the test for reference.
[MP+dmb.sttt+addr.litmus.txt](https://github.com/herd/herdtools7/files/10127417/MP%2Bdmb.sttt%2Baddr.litmus.txt)

Reply:
Good catch @maranget, I've removed it because we already have `MP+dmb.stTT+addr.litmus` and they are the same. I forced push the fix.

@jalglave the cat file passes the regressions (including ets2) even when we enumerate `tlbi-after` with no constraints (`0` instead of `hw-reqs`).